### PR TITLE
global: add translations support

### LINKED ISSUE #117: add i18n to details page and its components

### PR DESCRIPTION
**Pull request by kpsherva**
* closes #117

### CODE REVIEW

File: invenio_requests/assets/semantic-ui/translations/invenio_requests/scripts/compileCatalog.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
```
Comment (jrcastro2) [inline invenio_requests/assets/semantic-ui/translations/invenio_requests/scripts/compileCatalog.js:2]:
```suggestion
// This file is part of invenio-Requests
// Copyright (C) 2022 Graz University of Technology.
// Copyright (C) 2022 CERN
```
If we have this same code copy/pasted across different repositories maybe we could consider having it in an i18n repo, I see that we have [invenio-i18n](https://github.com/inveniosoftware/invenio-i18n), that could be a good candidate, WDYT?

File: invenio_requests/assets/semantic-ui/translations/invenio_requests/scripts/initCatalog.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
```
Comment (jrcastro2) [inline invenio_requests/assets/semantic-ui/translations/invenio_requests/scripts/initCatalog.js:2]:
```suggestion
// This file is part of Invenio-Requests
// Copyright (C) 2021 Graz University of Technology.
// Copyright (C) 2022 CERN.
```

Same question as above.

**Review by jrcastro2 — COMMENTED** [review]
LGTM!

I am missing a lot of context about the implementation of i18n but I left a question, sorry if is a silly one :sweat_smile:

File: invenio_requests/assets/semantic-ui/translations/invenio_requests/scripts/compileCatalog.js (hunk 1 above)
Comment (kpsherva) [reply invenio_requests/assets/semantic-ui/translations/invenio_requests/scripts/compileCatalog.js:2]: lets discuss on the standup, but I think it is a good idea

**Pull request merged**